### PR TITLE
325 access resources cli

### PR DESCRIPTION
**Pull request by serhiibets**

### CODE REVIEW

File: takeoff/takeoff_gui/lib/features/details/pages/resource_details.dart [hunk 1]
```diff
@@ -21,23 +20,23 @@ class ResourceDetails extends StatelessWidget {
         crossAxisAlignment: CrossAxisAlignment.start,
         children: [
           CustomButton(
-              text: AppLocalizations.of(context)!.openIdeButton,
```
Comment (capf4bbo) [inline takeoff/takeoff_gui/lib/features/details/pages/resource_details.dart:24]: I am not sure why you deleted this line

File: takeoff/takeoff_gui/lib/features/details/pages/resource_details.dart [hunk 2]
```diff
@@ -21,23 +20,23 @@ class ResourceDetails extends StatelessWidget {
...
         children: [
           CustomButton(
-              text: AppLocalizations.of(context)!.openIdeButton,
-              onPressed: () => controller.openResource(ResourceType.ide),
+              text: "Open IDE",
+              onPressed: () => controller.openResource(Resource.ide),
               icon: Icons.code),
           const SizedBox(height: 20),
           CustomButton(
-              text: AppLocalizations.of(context)!.openPipelineButton,
```
Comment (capf4bbo) [inline takeoff/takeoff_gui/lib/features/details/pages/resource_details.dart:29]: I am not sure why you deleted this line

File: takeoff/takeoff_gui/lib/features/details/pages/resource_details.dart [hunk 3]
```diff
@@ -21,23 +20,23 @@ class ResourceDetails extends StatelessWidget {
...
-              onPressed: () => controller.openResource(ResourceType.pipeline),
+              text: "Open Pipeline",
+              onPressed: () => controller.openResource(Resource.pipeline),
               icon: Icons.cloud_sync_outlined),
           const SizedBox(height: 20),
           CustomButton(
-              text: AppLocalizations.of(context)!.openFeRepo,
```
Comment (capf4bbo) [inline takeoff/takeoff_gui/lib/features/details/pages/resource_details.dart:34]: I am not sure why you deleted this line

File: takeoff/takeoff_gui/lib/features/details/pages/resource_details.dart [hunk 4]
```diff
@@ -21,23 +20,23 @@ class ResourceDetails extends StatelessWidget {
...
-              onPressed: () => controller.openResource(ResourceType.frontend),
+              text: "Open FE Repo",
+              onPressed: () => controller.openResource(Resource.feRepo),
               icon: Icons.account_tree_outlined),
           const SizedBox(height: 20),
           CustomButton(
-              text: AppLocalizations.of(context)!.openBeRepo,
```
Comment (capf4bbo) [inline takeoff/takeoff_gui/lib/features/details/pages/resource_details.dart:39]: I am not sure why you deleted this line

**Review by capf4bbo — CHANGES_REQUESTED** [review]

File: takeoff/takeoff_gui/lib/features/details/pages/resource_details.dart (hunk 1 above)
Comment (serhiibets) [reply takeoff/takeoff_gui/lib/features/details/pages/resource_details.dart:24]: Fixed.

File: takeoff/takeoff_gui/lib/features/details/pages/resource_details.dart (hunk 3 above)
Comment (serhiibets) [reply takeoff/takeoff_gui/lib/features/details/pages/resource_details.dart:34]: Fixed.

File: takeoff/takeoff_gui/lib/features/details/pages/resource_details.dart (hunk 4 above)
Comment (serhiibets) [reply takeoff/takeoff_gui/lib/features/details/pages/resource_details.dart:39]: Fixed.

File: takeoff/takeoff_gui/lib/features/details/pages/resource_details.dart (hunk 2 above)
Comment (serhiibets) [reply takeoff/takeoff_gui/lib/features/details/pages/resource_details.dart:29]: Fixed.

**Review by capf4bbo — APPROVED** [review]

**Pull request merged**